akka-remote/src/main/resources/reference.conf

```diff
@@ -1035,6 +1035,64 @@ akka {
         # it doesn't have to be thread-safe.
         # Refer to `akka.remote.artery.RemoteInstrument` for more information.
         instruments = ${?akka.remote.artery.advanced.instruments} []
+
+      }
+
+      # SSL configuration that is used when transport=tls-tcp.
+      ssl {
+        # Factory of SSLEngine.
+        # Must implement akka.remote.artery.SSLContext and have a public constructor with
+        # optional ActorSystem parameter.
+        ssl-engine-provider = akka.remote.artery.ConfigSSLEngineProvider
```

Comment:
:+1: Using the provider mechanism for the default makes sense. We could think about putting the settings for that default provider into its own section like we do in other similar cases.

Reply:
👍